tests/Testcontainers.Databases.Tests/Testcontainers.Databases.Tests.csproj

```diff
@@ -5,13 +5,59 @@
         <IsPublishable>false</IsPublishable>
     </PropertyGroup>
     <ItemGroup>
-        <PackageReference Include="Microsoft.NET.Test.Sdk"/>
-        <PackageReference Include="coverlet.collector"/>
-        <PackageReference Include="xunit.runner.visualstudio"/>
-        <PackageReference Include="xunit"/>
+        <PackageReference Include="Microsoft.NET.Test.Sdk" />
+        <PackageReference Include="coverlet.collector" />
+        <PackageReference Include="xunit.runner.visualstudio" />
+        <PackageReference Include="xunit" />
     </ItemGroup>
     <ItemGroup>
-        <ProjectReference Include="../Testcontainers.*.Tests/Testcontainers.*.Tests.csproj"/>
-        <ProjectReference Remove="Testcontainers.Databases.Tests.csproj"/>
+        <ProjectReference Include="..\..\..\testcontainers-dotnet\src\Testcontainers.Seq\Testcontainers.Seq.csproj" />
```

Comment:
Why changing the include mechanism that uses the `Testcontainers.*.Tests` wildcard?